README.md

```diff
@@ -54,6 +54,23 @@ allocated memory by file
 . . .
 ```
 
+Also you can print report to file. For this use `pretty_print` method with `to_file` option and `path_to_your_log_file` string:
+```
+$ pry
+pry> require 'memory_profiler'
+pry> MemoryProfiler.report(allow_files: 'rubygems'){ require 'mime-types' }.pretty_print(to_file: 'path_to_your_log_file')
```

Comment:
param 1 is IO so this example will not work maybe just allow it to take string_or_io in ... its less to type anyway.